doc/nrf/working_with_nrf/nrf52/developing.rst

```diff
@@ -34,17 +34,38 @@ You can also use FOTA updates to replace the application.
 FOTA over Bluetooth Low Energy
 ==============================
 
+FOTA updates are supported using MCUmanager's Simple Management Protocol (SMP) over Bluetooth.
+The application acts as a GATT server and allows the connected Bluetooth Central to perform the firmware update.
+Sample must support Bluetooth peripheral role (:kconfig:option:`CONFIG_BT_PERIPHERAL`) in order to use this feature.
+
+The application supports SMP handlers related to:
+
+* Image management.
+* Operating System (OS) management -- used to reboot the device after the firmware upload is complete.
+* Erasing settings partition -- used to ensure that a new application is not booted with incompatible content in the settings partition written by the previous application.
```

Comment:
If possible, I guess this section can be shared between the Thingy:53 ug and nRF52/53 DK UGs

Let's wait with this for the technical writers input

Reply:
I think so too. The only problem is the `CONFIG_BOOTLOADER_MCUBOOT` which is needed to be manually in nRF52/53 DKs and is already enabled by default fot Thingy:53

Reply:
So, if there is some part that is good for all of the listed docs (whether it is a whole section or just a subset of it), you can mark such area with the `.. fota_something_start` in the beginning and `.. fota_something_end` at the end of the area. Then just use that marked area in other docs the way you had in your earlier comment above.